src/kakarot/precompiles/blake2f.cairo

```diff
@@ -121,7 +122,7 @@ namespace Blake2 {
 
         // Compute state[14].
         local state14;
-        if (f == 1) {
+        if (f == TRUE) {
```

Comment:
actually it's better to test against FALSE, so can you switch the if/else : 

```
if (f == FALSE) {
    state14 = 0x1f83d9abfb41bd6b;
} else {
    state14 = 0xe07c265404be4294;
}
```

not that for this exact case, you case as well do

```
let state14 = (1 - f) * 0x1f83d9abfb41bd6b + f * 0xe07c265404be4294
```

Reply:
May I ask why is it better to test against FALSE?